src/announcement/announcement.ts

```diff
@@ -0,0 +1,81 @@
+//eslint-disable-next-line
+require("dotenv").config();
+import Web3 from "web3";
+import { Contract } from "web3-eth-contract";
+import { AbiItem } from "web3-utils";
+
+import { KeccakHash } from "../types/hash";
+import { hashPrefix } from "../utilities/hashPrefix";
+
+const BATCH_CONTRACT_ADDRESS = process.env.BATCH_CONTRACT_ADDRESS as string;
+
+export interface BatchArguments {
+  provider: Web3;
+  accountAddress: string;
+  uri: string;
+  hash: KeccakHash;
+}
+const BATCH_CONTRACT_ABI: AbiItem[] = [
+  {
+    anonymous: false,
+    inputs: [
+      {
+        indexed: false,
+        internalType: "bytes32",
+        name: "hash",
+        type: "bytes32",
+      },
+      {
+        indexed: false,
+        internalType: "string",
+        name: "dsnpUri",
+        type: "string",
+      },
+    ],
+    name: "DSNPBatch",
+    type: "event",
+  },
+  {
+    inputs: [
+      {
+        internalType: "bytes32",
+        name: "hash",
+        type: "bytes32",
+      },
+      {
+        internalType: "string",
+        name: "dsnpUri",
+        type: "string",
+      },
+    ],
+    name: "batch",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+];
+
+const contract: Contract | null = null;
+
+export const getContract = (web3Instance: Web3): Contract => {
```

Comment:
All exported methods should have a docblock. also we should have a lint that forces that...

Reply:
This method shouldn't have been exported. Will fix.